pkg/project/project.go

```diff
@@ -1,7 +1,7 @@
 package project
 
 var (
-	bundleVersion = "0.22.0"
+	bundleVersion = "0.21.1-dev"
```

Comment:
Why not having `0.22.0-dev` here and target master branch ?

Reply:
This is in line with https://github.com/giantswarm/giantswarm/blob/master/areas/empowerment/versioning-of-operators/how-to-release-an-operator.md#release-a-new-operator-version-to-all-installations.

I think bumping to patch makes most sense otherwise we would downgrade which is weird. Debatable.